packages/protractor/test/protractor-2/BUILD.bazel

```diff
@@ -41,7 +34,7 @@ ts_library(
     name = "ts_spec",
     testonly = True,
     srcs = ["test.spec.ts"],
-    tsconfig = ":tsconfig-test",
+    tsconfig = "//packages/typescript/test:tsconfig-test",
```

Comment:
this seems undesirable dependency between packages, why not have a copy of this tsconfig here if it's needed in protractor?

Reply:
👍